### PR TITLE
Added travis ci

### PR DESCRIPTION
**Pull request by Yashashree1997**
This PR solves #8 
I have enabled travis ci to run the linters on every PR.
Please have a look @chaitanyaenr

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+langauge: python
+
+python: "3.6"
+
+before_install:
+  - sudo apt-get update
+
+install:
+  - sudo apt install python3-pip 
+  - sudo pip3 install -r requirements.txt
```
Comment (chaitanyaenr) [inline .travis.yml]: This can be removed, let's just install the dependencies needed to run the linters which are in test-requirements.txt.

Comment (Yashashree1997) [reply .travis.yml]: Okay

**Comment by chaitanyaenr** [comment]
@Yashashree1997 can you add a link to the travis CI test build here please?

**Comment by Yashashree1997** [comment]
Link to Travis CI test build: https://travis-ci.org/github/Yashashree1997/cerberus/jobs/664833527

**Pull request merged**